protocol/communities/community_changes.go

```diff
@@ -122,19 +144,23 @@ func evaluateCommunityChangesByDescription(origin, modified *protobuf.CommunityD
 	// Check for new members at the org level
 	for pk, member := range modified.Members {
 		if _, ok := origin.Members[pk]; !ok {
-			if changes.MembersAdded == nil {
-				changes.MembersAdded = make(map[string]*protobuf.CommunityMember)
-			}
 			changes.MembersAdded[pk] = member
 		}
 	}
 
+	// Check ban/unban
+	findDiffInBannedMembers(modified.BannedMembers, origin.BannedMembers, changes.MembersBanned)
+	findDiffInBannedMembers(origin.BannedMembers, modified.BannedMembers, changes.MembersUnbanned)
+
+	// Check for new banned members (from deprecated BanList)
+	findDiffBanList(modified.BanList, origin.BanList, changes.MembersBanned)
```

Comment:
Just for conhesion, but this should probably be `findDiffInBanList` to use the same semantics as `findDiffInBannedMembers